doc/ecs/ecs_basics.md

```diff
@@ -0,0 +1,33 @@
+## Was ist ein ECS (Kurzform)
+
+**Entität:**
+Entitäten sind die Objekte im Spiel. Im Code sind sie nur leere Container, dessen Eigenschaften über die zugewiesenen Components bestimmt worden. Entitäten haben neben den Components keine eigenen Attribute oder Funktionen.
+
+**Component**
+Components sind die Datensätze der Entitäten. Jede Component-Instanz gehört zu genau einer Entität.
+Components beschreiben die Eigenschaften von Entitäten.
+Components speichern hauptsächlich Daten/den Zustand einer Entität.
+Teilweise kann es nötig sein, Logik in Components zu implementieren, dies sollte aber nur in Ausnahmefällen passieren und die implementierte Logik sollte möglichst klein gehalten werden.
+Siehe auch [Strategy Pattern im ECS](./ecs_and_strategy_pattern.md)
+
+**System**
+Systeme agieren auf Components und ändern die Werte in diesen. Sie beschreiben also das Verhalten der Entitäten. Ein System kann auf ein oder mehrere Components agieren.
+In Systemen wird die eigentliche Logik implementiert.
+
+Eine Entität und dessen Verhalten wird also daher bestimmt, welche Components es hat und wie die Systeme mit dieser Component-Kombination arbeiten.
+
+## Basisstruktur
+
+[ECS](./img/ecs.png)
+
+Neu erzeugte Entitäten speichern sich automatisch im HashSet `entities` der `ECS`-Klasse ab.
```

Comment:
Hier sind mehrere Punkte unklar:

1. Warum speicherung in `ECS`? 
2. Was heisst "automatisch"? 
3. Wieso ein HashSet? Das ist ein Implementierungsdetail - das Attribut ist sogar auf "+" (public)?
4. Wenn sich die Entitäten in der _Klasse_ speichern - ist die dann irgendwie statisch oder so?
5. Welche Rolle hat die Klasse?